source/code/systems/asset_system/private/asset_storage.cxx

```diff
@@ -101,17 +161,304 @@ namespace ice
     public:
         DefaultAssetStorage(
             ice::Allocator& alloc,
-            ice::ResourceTracker& resource_tracker,
-            ice::UniquePtr<ice::AssetTypeArchive> asset_archive
+            ice::UniquePtr<ice::AssetTypeArchive> asset_archive,
+            ice::AssetStorageCreateInfo const& create_info
         ) noexcept;
 
         ~DefaultAssetStorage() noexcept override;
 
-        auto request(
+        auto bind(
             ice::AssetType type,
-            ice::String name,
+            ice::String name
+        ) noexcept -> ice::Asset
```

Comment:
Create a brainstorming task to see if the 'bind' method of asset system can be changed

Reply:
Created issue: https://github.com/iceshard-engine/engine/issues/159